pipeline/outputs/elasticsearch.md

```diff
@@ -21,6 +21,7 @@ The **es** output plugin, allows to ingest your records into an [Elasticsearch](
 | AWS\_STS\_Endpoint | Specify the custom sts endpoint to be used with STS API for Amazon OpenSearch Service |  |
 | AWS\_Role\_ARN | AWS IAM Role to assume to put records to your Amazon cluster |  |
 | AWS\_External\_ID | External ID for the AWS IAM Role specified with `aws_role_arn` |  |
+| AWS\_Service\_Name | Service name to be used in AWS Sigv4 signature. For integration with Amazon OpenSearch Serverless, set to `aoss`. See the [FAQ](opensearch.md#faq) section on Amazon OpenSearch Serverless for more information. | es |
```

Comment:
I'm worried this relative FAQ won't work but I guess it does...

Reply:
I think it does work. I tried it in vscode and the reference was found. In the diff, it takes me to `https://github.com/fluent/fluent-bit-docs/blob/aoss/pipeline/outputs/opensearch.md`